R/utils-classes.R

```diff
@@ -243,13 +236,18 @@ collect.bcdc_promise <- function(x, ...){
       limit_param = "count",
       offset_param = "startIndex",
       limit = number_of_records,
-      limit_chunk = 5000,
+      limit_chunk = 1000,
```

Comment:
Why the smaller limit chunk?

Reply:
My thought (and I am very open to feedback) was to go conservative. I had greater success with lower chunk limits on lower bandwidth connections.